download.sh

```diff
@@ -62,9 +62,10 @@ do
     wget --continue ${PRESIGNED_URL/'*'/"${MODEL_PATH}/params.json"} -O ${TARGET_FOLDER}"/${MODEL_PATH}/params.json"
     wget --continue ${PRESIGNED_URL/'*'/"${MODEL_PATH}/checklist.chk"} -O ${TARGET_FOLDER}"/${MODEL_PATH}/checklist.chk"
     echo "Checking checksums"
-    if [ "$CPU_ARCH" = "arm64" ]; then
+    CPU_ARCH=$(uname -m)
```

Comment:
Why is this necessary? `CPU_ARCH=$(uname -m)` was set on line 25 once already.

Reply:
Oh, I didn't see it, thank you!